notes/ltc-cola-summary.vbs

```diff
@@ -518,13 +522,13 @@ If income_summary_check = 1 then
       Text 5, 160, 45, 10, "Other notes:"
       Text 135, 175, 60, 15, "Worker signature:"
     EndDialog
-    
-    Do 
+
```

Comment:
Same as other dialog: worker_signature is in dialog, but not mandatory in the do....loop. Either remove variable from dialog or add handling and error messaging.